views/index-copy.ejs

```diff
@@ -0,0 +1,125 @@
+<% include partials/header %>
+
```

Comment:
One tip regarding comments in the HTML: I have seen people add a comment to the closing tag only, to make it clearer which element is being closed. This helps avoid “start of” comments which are often redundant because the class or id of the element is right after the comment.